style/sign-in.css

```
@@ -1,202 +1,281 @@
 
-.login-container {
-    position: absolute; 
-    width: 640px;
-    height: 618px;
-    top: 231px;
-    left: 640px;
-    gap: 40px;
-    
-    
-    
+    html {
+      font-size: 16px;
+      box-sizing: border-box;
   }
-  .title{
-    display: flex;
-    width: 396px;
-    height: 132px;
-    align-items: center;
-    justify-content: center; 
-    margin: 0 122px;
-    gap:10px;
-    margin-bottom: 40px;
-   
-    
-   
+  
+  .login-container {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      max-width: 40rem;
+      width:40rem;
+     margin: 14.4375rem auto;
+      height: 38.625rem;
+     /* gap:2.5rem; */
   }
-  .title img{
-    width: 103.53px;
-    height: 103.88px;
-    top: 12.98px; 
-   
-  }
-  .title h1{
-    width:266px;
-    height: 90px;
-    top:23.98px;
-    left: 125.77px;
-    font-family: ROKAF Sans;
-    font-weight: 700;
-    font-size: 66.34px;
-    line-height: 89.56px;
-    letter-spacing: 0%;
-    color:#3692FF;
- 
+  
+  
+  .title {
+      display: flex;
+      width: 24.75rem;
+      height: 8.25rem;
+      align-items: center;
+      justify-content: center;
+      gap: 0.5rem;
+      margin-bottom: 1.5rem;
+      
+      
+      
   }
-  .input-form{
-    width: 640px;
-    height: 300px;
-    gap: 24px;
-    margin-top:40px;
+  
+  .title img {
+      width: 6.47rem;
+      height: 6.49rem;
+     
   }
-  .group{
-    width: 100%;
-    height: 98px;
-    margin-bottom: 16px;
-    
+  
+  .title h1 {
+      width: 16.625rem;
+      height: 6rem;
+      left: 7.86rem;
+      font-family: ROKAF Sans;
+      font-weight: 700;
+      font-size: 4.15rem;
+      line-height: 5.6rem;
+      letter-spacing: 0%;
+      color: #3692FF;
+     
   }
-  .group label{
-    width: 47px;
-    height: 26px;
-    font-family: Pretendard;
-    font-weight: 700;
-    font-size: 18px;
-    line-height: 26px;
-    letter-spacing: 0%;
-
+  
+  .input-form {
+      width: 40rem;
+      height: 18.75rem;
+      gap: 1.5rem;
+      margin-top: 2.5rem;
   }
- 
-  .group input{
-    display: block;
-    width: 632.68px;
-    height: 51.67px;
-    gap: 10px;
-    border-radius: 12px;
-    background-color: #F3F4F6;
-    border: 2px;
-    margin-top: 16px;
-    
+  
+  .group {
+      width: 100%;
+      height: 6.125rem;
+      margin-bottom: 1rem;
+  }
+  
+  .group label {
+      width: 2.9375rem;
+      height: 1.625rem;
+      font-family: Pretendard;
+      font-weight: 700;
+      font-size: 1.125rem;
+      line-height: 1.625rem;
+      letter-spacing: 0%;
+  }
+  
+  .group input {
+      display: block;
+      width: 100%;
+      height: 3.5rem;
+      gap: 0.625rem;
+      border-radius: 0.75rem;
+      background-color: #F3F4F6;
+      border: 2px;
```

💊 제안
border라는 css 속성은 단축 속성으로 실제로는 border-width, border-style, border-color 속성을 하나의 속성으로 적는 것입니다. 지금의 경우 border-width만 작성되어 있으니 이런경우 border-width로 적어주시는 것이 더 명확하고 가독성 측면에서도 좋습니다. 만약 다른 속성을 적지 않으신 거라면 적어주시는 것을 추천드려요!

```suggestion
      border-width: 2px; // border-width만 필요할 경우 이렇게
      border: 2px solid var(--blue); // border를 적을경우 모든 속성을 기입
```

[mdn border](https://developer.mozilla.org/ko/docs/Web/CSS/border)